shared/utils/enums.py

```diff
@@ -43,4 +43,5 @@ class TaskConfigGroup(Enum):
     synchronize = "synchronize"
     timeseries = "timeseries"
     upload = "upload"
+    upload_test_result = "upload_test_result"
```

Comment:
Maybe just `test_results`?  All the test result ingestion tasks can be in that same group.  I'm using `app.tasks.bundle_analysis.<TaskName>` for bundle analysis stuff.